### ISSUE TITLE
Discussion and potential first work towards verb conjugation on the website

### ISSUE THREAD

**Issue by andrewtavis** [scribe-org/scri.be#2]
### Terms

- [X] I have searched [open and closed issues](https://github.com/scribe-org/scri.be/issues?q=is%3Aissue)
- [X] I agree to follow Scribe's [Code of Conduct](https://github.com/scribe-org/scri.be/blob/main/.github/CODE_OF_CONDUCT.md)

### Issue

As we plan to make the Scribe-Conjugate app series at some point with the data from Scribe-Data and Scribe-Server, maybe it would make sense to create a `conjugate.scri.be` URL at some point that would be a web version of the app. This would doubtless drive attention to the community and the projects in general, and could lead to more conjugations being added to Wikidata 😊

**Comment by akshayw1** [comment]
Hey @andrewtavis @angrezichatterbox Intrested to work over these

**Comment by andrewtavis** [comment]
Yes this would be really cool to look into, @akshayw1! Let's keep this in mind. I hope to have the website up this weekend finally as there's been a ton going on the last few weeks. From there we can maybe work on designs for this. Might make sense that we leave this for after GSoC though as there's enough to do in the other projects? I don't us to spread ourselves too thin 🤔

**Comment by andrewtavis** [comment]
Discussing in the sync, we can start this when the website is released :)
